src/components/NewTodo.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { KEYCODE } from '../utils/const';
+
+export class NewTodo extends React.PureComponent {
+  state = {
+    inputValue: '',
+  };
+
+  handleGlobalClick = (event) => {
+    if (event.target.closest('.js-new-todo')) {
```

Comment:
no dom methods in react

Reply:
as discussed - I removed this functionality that saves new todo on outside click